alts/src/main/java/io/grpc/alts/AltsChannelBuilder.java

```diff
@@ -45,6 +46,7 @@
  * ALTS version of {@code ManagedChannelBuilder}. This class sets up a secure and authenticated
  * commmunication between two cloud VMs using ALTS.
  */
+@ExperimentalApi
```

Comment:
We typically make a tracking issue for each usage of ExperimentalApi,    You can see it in other usages of the annotation.  It allows us to keep track of when to make it non experimental.

Reply:
Added a tracking issue.